src/Umbraco.Forms.Integrations.Automation.Zapier/Services/ZapierFormSubscriptionHookService.cs

```diff
@@ -38,18 +37,15 @@ public ZapierFormSubscriptionHookService(IScopeProvider scopeProvider, ILogger l
         }
 #endif
 
-        public bool TryGetById(string id, out IEnumerable<FormConfigDto> dto)
+        public bool TryGetById(string id, out IEnumerable<string> dto)
         {
             using (var scope = _scopeProvider.CreateScope())
             {
                 var entities =
                     scope.Database
-                        .Query<FormConfigDto>( "SELECT * FROM zapierSubscriptionHook where EntityId = @0", id)
-                        .ToList();
+                        .Query<string>( "SELECT HookUrl FROM zapierSubscriptionHook where EntityId=@0", id).ToArray();
```

Comment:
This should have `and Type = 2` (ideally reading the 2 from a constant).

Reply:
Updated.